README.md

```diff
@@ -1,3 +1,5 @@
+[![Java CI](https://github.com/michaelmior/xeger/actions/workflows/ci.yml/badge.svg)](https://github.com/michaelmior/xeger/actions/workflows/ci.yml)
```

Comment:
This should be changed to `https://github.com/bluezio/xeger`.

Reply:
Good catch! Fixed.

Reply:
Thanks!